files/deployment.yaml

```diff
@@ -29,52 +29,71 @@
       command: oc whoami -t
       register: kubeconfig_token
       become: "{{ as_root }}"
+
+    # path_to_secrets = this is what we get from the outside
+    # internal_secrets_path = this is what we'll use here
```

Comment:
pardon my ignorance but where here? (I don't see it anywhere)

Reply:
/facepalm

Reply:
corrected it now and even improved the logic around dirs